lib/matplotlib/colors.py

```diff
@@ -845,6 +845,12 @@ def color_block(color):
                 f'over {color_block(self.get_over())}'
                 '</div>')
 
+    def copy(self):
+        """
+        Return a copy of the colormap.
+        """
```

Comment:
```suggestion
        """Return a copy of the colormap."""
```
If it fits on one line, we only use one line.